src/app/metadata/features/dataset-summary/dataset-summary.badges.component.html

```diff
@@ -0,0 +1,29 @@
+<ul class="ps-5">
+  @for (
+    item of data();
+    track item.value;
+    let idx = $index;
+    let count = $count;
+    let last = $last
+  ) {
+    <li>
+      <mat-chip class="small-chips mb-0.5 text-xs"
+        ><span class="uppercase">{{ item.value | underscoreToSpace }} </span>
+        <strong>{{ item.count }}</strong></mat-chip
+      >
+    </li>
+    @if (count > 3) {
```

Comment:
Please consider the edge cases. As far as I see, if you have exactly 4 items, you will see the button, but it will do nothing except changing its label.

Reply:
Would be nice if the button not just says "see more", but "see N more" where N is the number of hidden items. Or if it just says "see all" since it doesn't work gradually anyway.

Reply:
Thanks. It was a classic off-by-one.